packages/ir-sdk/fern/apis/ir-types-latest/changelog/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v62.0.0] - 2025-11-04
+
+- **BREAKING**: Add file upload examples support to ExampleRequestBody with fileUpload variant. This is a breaking change because adding a new union variant requires all generators to handle it.
```

Comment:
For consistency we could just call it `Feature: ...`

I do like the rest of the name though, it's nice to say that it's a breaking change